modules/bitgo/package.json

```diff
@@ -103,7 +103,7 @@
     "karma-mocha": "^2.0.1",
     "karma-typescript": "^5.0.3",
     "minimist": "^1.2.5",
-    "nock": "=9.0.28",
+    
```

Comment:
NIT: but this shouldn't be an empty blank line in the middle of our dependencies

Reply:
This is removed in the latest commit.